app/assets/javascripts/components/overview/my_articles.jsx

```diff
@@ -62,6 +63,7 @@ const MyArticles = React.createClass({
               student={this.props.current_user}
               assignments={reviewing}
               prefix={I18n.t('users.my_reviewing')}
+              tooltip_message="Assign yourself an article to review"
```

Comment:
`This if for keeping track of which article(s) you plan to peer review or provide feedback on.`